M64MM2/AboutForm.ru.resx

```diff
@@ -287,7 +287,7 @@
 - Onewingdangl (создатель SM64MM 1.0)
 - James Pelster (CaptainSwag101, создатель M64MM2.0)
 - Dudaw (за числа для смен анимаций)
-- yoshielectron (документация чит-кодов для SM64)
+- yoshielectron (за документацию чит-кодов для СМ64)
```

Comment:
Oh, you guys don't use the English title verbatim?

Reply:
Not sure if that's the case with the entire Russian-speaking SM64 community, but in the machinima part of the community it's way more common to see "СМ64" being used instead of "SM64," and I decided to change it to that in yoshielectron's credit since I also changed it in the beginning ("Эта программа предназначена для сообщества машиним по **СМ64**.").